openshift/scripts/oc_deploy.sh

```diff
@@ -21,13 +21,20 @@ source "$(dirname ${0})/common/common"
 #%   ${THIS_FILE} pr-0 apply
 #%
 
+# Use keycloak prod url when deploying to prod
+if [ "${SUFFIX}" = "prod" ]; then
+    KEYCLOAK_AUTH_URL=${KEYCLOAK_PROD_URL}
+else
+    KEYCLOAK_AUTH_URL=${KEYCLOAK_DEV_URL}
+fi
+
 # Target project override for Dev or Prod deployments
 #
 PROJ_TARGET="${PROJ_TARGET:-${PROJ_DEV}}"
 
 # Process a template (mostly variable substition)
 #
-OC_PROCESS="oc -n ${PROJ_TOOLS} process -f ${PATH_DC} -p NAME=${NAME_APP} -p SUFFIX=${SUFFIX}"
+OC_PROCESS="oc -n ${PROJ_TOOLS} process -f ${PATH_DC} -p NAME=${NAME_APP} -p SUFFIX=${SUFFIX} -p KEYCLOAK_AUTH_URL=${KEYCLOAK_AUTH_URL}"
```

Comment:
Please remove this - sorry for asking for it yesterday - I was totally wrong - this doesn't have to be passed to the template - we need a global configmap in our dev project that has all of these things specified.